src/execution/riscv64/simulator-riscv64.h

```diff
@@ -693,6 +786,10 @@ class Simulator : public SimulatorBase {
   // Floating-point control and status register.
   uint32_t FCSR_;
 
+  //RVV registers
+  __int128_t Vregister_[kNumVRegisters];
+  uint64_t vstart_, vxsat_, vxrm_, vcsr_, vtype_, vl_, vlenb_;
```

Comment:
Is vtype should have the uint128_t type? According to "3.3 vector type register" at rvv1.0 spec page11?

Reply:
The read-only XLEN-wide vector type CSR

It mean it is 64-bit in rv64

Reply:
All right!